activities/activity_nick_grokhowsky.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+# Function to show significantly high and low pixels 
+def run_highestPixelValues(elev, env, **kwargs):
+	# Create a smooth surface from the elevation LiDAR points
+    gs.run_command(
+        'r.neighbors',
+         input=elev,
+         output='smooth',
+         method='average',
+         flags='c',
+	 env=env
+    )
+    # Calculate univariate statistics
+    stats = gs.parse_command('r.univar', map='smooth', env=env)
+    
+    # Parse the stats library and capture the values for mean and standard deviation
+    for value in stats:
+        if 'mean' in value:
+            mean = float(value[ -7: ])
+        if 'standard deviation' in value:
+            sd = float(value[ -7: ])
+    high = str(mean + sd)
+    low = str(mean - sd)
+
+    # Using map algebra create a new raster map of highest and lowest pixel values versus all others
+    gs.mapcalc('high_values=if(smooth >= ' + high + ')') 
+    gs.mapcalc('new_values=high_values + 1') 
+    gs.mapcalc('low_values=if(smooth > ' + low + ')') 
+    gs.mapcalc('final_values=high_values + low_values') 
+
+    # Change colors for high and low maps
+    gs.run_command('r.colors', map='high_values', color='elevation', env=env)
+    gs.run_command('r.colors', map='low_values', color='elevation', env=env)
+    gs.run_command('r.colors', map='new_values', color='elevation', env=env)
+    gs.run_command('r.colors', map='final_values', color='elevation', env=env)
```

Comment:
Maybe use different color for each, depending on what you are trying to do.

Reply:
I am going to only change the color for the 'final_values' in the next iteration because that is the primary output I am after.  The other raster images are just means to this end.